parameters.cpp

```diff
@@ -157,6 +157,7 @@ Realf P::vamrCoarsenLimit = 0.5;
 string P::vamrVelRefCriterion = string("");
 
 uint P::amrMaxSpatialRefLevel = 0;
+uint P::amrMaxAllowedSpatialRefLevel = 0;
```

Comment:
Rather have this as a signed integer, with e.g. negative values for automatic behavior

Reply:
Ah, *this* is why it wasn't merged yet. :)

Reply:
Better?

Reply:
Perfect